src/main/webapp/resources/js/pages/user/components/UserAccountLayout.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { Outlet } from "react-router-dom";
+import { Col, Row } from 'antd';
+import { PageWrapper } from "../../../components/page/PageWrapper";
+import UserAccountNav from "./UserAccountNav";
+
+/**
+ * React component that layouts the user account page.
+ * @returns {*}
+ * @constructor
+ */
+export default function UserAccountLayout() {
+  return (
+    <PageWrapper title={i18n("UserAccountLayout.page.title")}>
```

Comment:
Add a back button that takes you back to the user listing

Reply:
We kinda talked about this one earlier. I can do a simple navigate(-1), but if the user navigates through the tabs, it we go back through the tab history. I can make it so that there's only a back button if the logged in user is an admin, but there if the admin get to this page from the main navigation bar, I dunno if that makes sense.

Reply:
Yup that makes sense and should work great

Reply:
Please see 3c1a7f20243b2e997175a247efa260848b471dab.